src/ResponseCache.php

```diff
@@ -90,4 +90,23 @@ protected function addCachedHeader(Response $response): Response
 
         return $clonedResponse;
     }
+
+    /**
+     * @param string|array $uris
+     */
+    public function forget($uris): self
+    {
+        $uris = is_array($uris) ? $uris : func_get_args();
+        
+        foreach ($uris as $uri) {
```

Comment:
Could you rewrite this use a collection (`filter` and `each` operation).

Reply:
I'm able to refactor to this:

```php
collect($uris)->each(function ($uri) {
    $request = Request::create($uri);
    $hash = $this->hasher->getHashFor($request);

    if ($this->cache->has($hash)) {
        $this->cache->forget($hash);
    }
});
```

But how can I improve that with a filter ?

Reply:
`filter` might be overkill, and not handy here indeed, let's leave it like this.